runtime/worker.rs

```diff
@@ -83,6 +84,32 @@ pub struct WorkerOptions {
   pub stdio: Stdio,
 }
 
+impl Default for WorkerOptions {
+    fn default() -> Self {
+        Self {
+            web_worker_preload_module_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
+            create_web_worker_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
+            module_loader: Rc::new(FsModuleLoader),
```

Comment:
Currently a module loader is required. It might be possible to just transform this field into an `Option` and use `Default::default()` here, then do the translation to the default loader further down the line. But I think the `FsModuleLoader` should work fine for most and `deno_core` will use a default loader anyway if it's not specified.